xlml/apis/task.py

```diff
@@ -669,6 +669,52 @@ def run(self) -> DAGNode:
 
     return group
 
+  def fullrun(self) -> DAGNode:
```

Comment:
What's the difference here from `run`? Just that metrics are handled properly? If so, please just merge that change into `run`. I told @RissyRan I'd implement those anyway :sweat_smile:

Reply:
> Yes, we miss post_process for it, but I see the test is green for metric processing?

Because I am calling `fullrun` to handle the metric processing in this case. `fullrun` contains the postprocess.

Reply:
> What's the difference here from `run`? Just that metrics are handled properly? If so, please just merge that change into `run`. I told @RissyRan I'd implement those anyway 😅

If we add postprocess into `run`, then all the nightly test will contains the postprocess step. However, we don't need postprocess for nightly tests. Do we still want to unify those runs?

Reply:
Based on the offline discussion, I unified all gke run to use `run` function.